.github/workflows/members.json

```diff
@@ -0,0 +1,233 @@
+[
```

Comment:
Wait, this is the approved workflow? We're supposed to manually maintain this file?

Is there a way to just add `skaffold-team` to this file? I don't even know who most of these people are

Reply:
(And skaffold-team has the advantage of being automatically synced to our internal systems)